code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -179,7 +179,7 @@
 				to_chat(src, "<i>There are too many of us...</i>")
 				return
 			var/list/babies = list()
-			var/new_nutrition = round(nutrition * 0.9)
+			var/new_nutrition = 700
```

Comment:
magic numbers bad, maybe use a define?

Reply:
Hashed out via discord, and I now understand what was being requested of me.